main.py

```diff
@@ -3695,10 +3695,58 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                     config["promoschanged"] = "yes"
                     cur.execute("REPLACE INTO config(name, value) VALUES('promoschanged', 'yes')")
                 return
+            if command == "freepacks" or (command == "bet" and len(args) > 0 and args[0].lower() == "packs"):
```

Comment:
This just occurred to me. You may want to alias this to `freepack` as well to account for people thinking "I'll open *a* free pack", similar how you have a singular version for `!sets` with `!set`.